docs/standard/frameworks.md

```diff
@@ -14,139 +14,150 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
-|                            |              | net47                                        |
-| Windows Store              | netcore      | netcore [netcore45]                          |
-|                            |              | netcore45 [win, win8]                        |
-|                            |              | netcore451 [win81]                           |
-| .NET Micro Framework       | netmf        | netmf                                        |
-| Silverlight                | sl           | sl4                                          |
-|                            |              | sl5                                          |
-| Windows Phone              | wp           | wp [wp7]                                     |
-|                            |              | wp7                                          |
-|                            |              | wp75                                         |
-|                            |              | wp8                                          |
-|                            |              | wp81                                         |
-|                            |              | wpa81                                        |
-| Universal Windows Platform | uap          | uap [uap10.0]                                |
-|                            |              | uap10.0 [win10] [netcore50]                  |
-
-## Deprecated frameworks
-
-The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
-
-| Deprecated framework | Replacement |
-| -------------------- | ----------- |
-| aspnet50             | netcoreapp  |
-| aspnetcore50         |             |
-| dnxcore50            |             |
-| dnx                  |             |
-| dnx45                |             |
-| dnx451               |             |
-| dnx452               |             |
-| dotnet               | netstandard |
-| dotnet50             |             |
-| dotnet51             |             |
-| dotnet52             |             |
-| dotnet53             |             |
-| dotnet54             |             |
-| dotnet55             |             |
-| dotnet56             |             |
-| netcore50            | uap10.0     |
-| win                  | netcore45   |
-| win8                 | netcore45   |
-| win81                | netcore451  |
-| win10                | uap10.0     |
-| winrt                | netcore45   |
+When you target one or more frameworks in an app or library, you're specifying the sets and versions of APIs that you'd like to make available to the app or library. You specify target frameworks in the project file using Target Framework Monikers (TFMs). When you specify multiple target frameworks, you may conditionally reference assemblies for each target framework. In your code, you can conditionally compile against those assemblies by using preprocessor symbols with *if-then-else* logic.
```

Comment:
My goal was to explain what the topic is about in the first paragraph: It's a summary of the main points of the topic. Unfortunately, its not helpful to my case that topics don't have a *Summary* section to make it clear that that's what the opening paragraph is trying to do here; therefore, any topic that tries to do this (and there are a bunch, of course) might result in having lines in the opening paragraph that look a little out of place.

Although a one-sentence opening paragraph is ok technically, I'm not a big fan because then paragraphs end up looking like bullet points with no 🍖 meat on the bone ... the idea is: Can't we find something useful/helpful/important to say to the reader on their second contact with the topic? ... right after reading the title. Can we at least help a bunch of readers looking for some specific info by just mentioning some of the main important points covered?

At least in this format, the reader knows a few things right off the bat about this topic's coverage from sentences 2 and 3:

* important term: TFM
* the project file is where you set these things
* you can have multiple target frameworks
* you can conditionally set assemblies on them
* you can conditionally compile code on them

I'll look closer after I get back from the :muscle: gym. Maybe a 3rd idea for the opening para will hit. :zap: